README.md

```diff
@@ -100,10 +100,139 @@ This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0
+above, and then not adding any other conda packages to it.  This
+is similar to the best practice usage of not adding any conda
+packages to your 'base' conda environment.
+
+Instead, when you need to have a conda environment with other
+packages that also needs to be able to run ISIS programs, we have
+two different options.  In both cases, we'll assume that you create
+a new environment called 'working' (but it could be named anything)
+that you want to add some conda packages to, but from which you
+also want ISIS access.
+
+The first step is to create 'working' and add whatever conda packages you want.
+
+#### Easy mode, with stacking
+
+1.  `conda activate isis`
+
+2.  `conda activate --stack working`
+
+That's it.  Told you it was easy.
+
+This activates the isis environment, gets it all set up, and
+then it 'stacks' the new working environment on top of it.  To get
+out, you'll have to `conda deactivate` two times to get out of
+`working` and then out of `isis`.
+
+
+#### Harder mode, with activation script hacking
+
+The above stacking situation may have issues if you have a particularly
+complicated set of packages or other dependencies.  The idea here is that
+the only thing you *really* need in your 'working' environment are
+the ISIS environment variables and the path to the ISIS executables.
+
+If the above paragraph sounded like gibberish, please seek help
+from your system administrator or local computer guru.
+
+And we can do this via customizations in the conda environment's
+activate.d/ and deactivate.d/ directories.  Adding these things can
+also be done manually from the command line, but encoding them in
+the activate.d/ and deactivate.d/ scripts is handy.
+
+1.  Create your conda environment however you like, adding whatever
+packages you need.  If you were reading the directions above, you've
+already done this.
+
+2.  Locate the path to your conda environments:
+
+        conda activate
+        echo $CONDA_PREFIX
+        conda deactivate
+
+    You'll probably get a directory that is in your home directory and
+    is named `anaconda3` or `miniconda3` or something similar.  For the
+    rest of this set of instructions, we'll refer to it as `$HOME/anaconda3`
+    to represent a directory named `anaconda3` in your home directory, but
+    this should be whatever you get from the above echo command.
+
+3.  Locate the path to your ISIS conda environment:
+
+        conda activate isis
+        echo $CONDA_PREFIX
+        conda deactivate
+
+    This should probably be `$HOME/anaconda3/envs/isis`.  You can see
+    that it starts with whatever you got from step 1, and ends in the
+    name of your isis environment, if you followed the installation
+    instructions above, you called that environment 'isis'.
+    
+    You can do the same thing to find the path to your new 'working'
+    environment, but in this example, it will be at
+    `$HOME/anaconda3/envs/working`.
+
```

Comment:
:+1: This looks really good for the uninitiated in how Conda is storing envs.

Reply:
Thanks.  I was new to conda just a year ago, and honestly, 24h ago, I didn't even know about the --stack option.